packages/instantsearch-ui-components/src/components/chat/ChatMessages.tsx

```diff
@@ -94,7 +94,7 @@ export type ChatMessagesProps<
   /**
    * Tools available for the assistant
    */
-  tools?: ClientSideTools;
+  tools: ClientSideTools;
```

Comment:
I know it doesn't matter too much since this is internal, but tools are technically optional since agents don't necessarily need to have them added.

Reply:
yes it's optional, but then you just pass an empty object, no?